ksqldb-rest-app/src/test/java/io/confluent/ksql/api/CorsTest.java

```diff
@@ -251,7 +251,7 @@ private void shouldAcceptPreflightRequest(final String origin, final String allo
         MultiMap.caseInsensitiveMultiMap().add("origin", origin)
             .add(ACCESS_CONTROL_REQUEST_METHOD_HEADER, "POST"));
 
-    assertThat(response.statusCode(), is(200));
+    assertThat(response.statusCode(), is(204));
```

Comment:
I couldn't track down what made this change, but it seems like a refinement on 200 so figured it's OK

Reply:
We're going to need a system test update when this change gets deployed: https://github.com/confluentinc/cc-system-tests-ksqldb/blob/953ba5df208ccd7cc2c3e4011465b81ecc9e69a7/test/ksql/ksql_test.go#L749